guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -111,16 +111,23 @@ To view the status of the cache, enter
 
 A sample follows:
 
+There are 13 types of cache types display as result for Magento open source by default.
```

Comment:
It's generally not a good idea to say how many items there are. This gets overlooked when a new item gets added. Delete this sentence.